docs/src/main/asciidoc/security-keycloak-authorization.adoc

```diff
@@ -393,18 +396,19 @@ quarkus.keycloak.devservices.resource-aliases.policies=/policies.jar <1>
 # Map the policies archive to a specific location in the container
 quarkus.keycloak.devservices.resource-mappings.policies=/opt/keycloak/providers/policies.jar <2>
 ----
-<1> Creates a `policies` alias for the `/policies.jar` classpath resource. The policies archive can also be located on the file system.
+<1> Creates a `policies` alias for the `/policies.jar` classpath resource.
+The policies archive can also be located on the file system.
```

Comment:
IMHO splitting a simple bullet message into several lines is not strictly necessary.

Reply:
Personally, I also prefer avoiding hard breaks between sentences. However, I follow this approach as it’s requested by our documentation team.